pages/common/autoflake.md

```diff
@@ -0,0 +1,19 @@
+# autoflake
+
+> A tool to remove unused imports and variables from Python code.
+
+- Remove unused variables from a single file and display diff:
```

Comment:
Perhaps ...`a diff` would read better here?

Reply:
How about `the diff`?

Reply:
Sure!

Reply:
👍  Updated